### PR TITLE
Eswe 619 create kotlin model for json entity part2

### PR DESCRIPTION
**Pull request by sacintha**

### CODE REVIEW

File: src/main/resources/migration/V1_1__start.sql [hunk 1]
```diff
@@ -1,13 +1,106 @@
...
+
+create sequence "article_seq" start with 1 increment by 50;
+
+create sequence "goal_steps_seq" start with 1 increment by 50;
+
+create sequence "profile_goals_seq" start with 1 increment by 50;
+
+create table "functional"
+(
+    "ciagprofile_offender_id"     varchar(255) not null,
```
Comment (theDustRoom) [inline src/main/resources/migration/V1_1__start.sql:29]: Why do we need another id when we have `offender_id`?

File: src/main/resources/migration/V1_1__start.sql [hunk 2]
```diff
@@ -1,13 +1,106 @@
...
+    "timeperiod" smallint,
+    "goals_id"   bigint,
+    primary key ("id")
+);
+
+create table "profile_goals"
+(
+    "id"                  bigint not null,
+    "goal"                varchar(255),
+    "profile_offender_id" varchar(255),
```
Comment (theDustRoom) [inline src/main/resources/migration/V1_1__start.sql:89]: Same here, why is this not just `offender_id`?

**Review by theDustRoom — CHANGES_REQUESTED** [review]

File: src/main/resources/migration/V1_1__start.sql (hunk 1 above)
Comment (sacintha) [reply src/main/resources/migration/V1_1__start.sql]: Sorry this SQl was generated as a first round of test.Now have updated.

File: src/main/resources/migration/V1_1__start.sql (hunk 2 above)
Comment (sacintha) [reply src/main/resources/migration/V1_1__start.sql]: Sorry this SQl was generated as a first round of test.Now have updated.

**Review by theDustRoom — APPROVED** [review]
LGTM

**Pull request merged**